README.md

```diff
@@ -80,11 +80,21 @@ You are encouraged to also create your own library of small, reusable functions
       hobbies: string;
       address: {
         city: string;
-        zip: number;
+        zip: string;
+      };
+    }
+
+    interface IRecord {
+      job: string;
+      name: string;
+      hobbies: string;
+      address: {
+        city: string;
+        zip: string | null;
```

Comment:
it would be good for something in this example to be a number (anything other than a string). zip? Or a new property?